var/spack/repos/builtin/packages/topaz/package.py

```diff
@@ -11,18 +11,21 @@ class Topaz(PythonPackage):
     featuring micrograph and tomogram denoising with DNNs."""
 
     homepage = "https://topaz-em.readthedocs.io/"
-    pypi = "topaz-em/topaz-em-0.2.5.tar.gz"
+    pypi = "topaz-em/topaz_em-0.2.5.tar.gz"
```

Comment:
This change breaks the ability to `spack checksum` `0.2.5`. 

One option, though I don't recall if it is kosher per @adamjstewart, is to add the url argument (`url="https://files.pythonhosted.org/packages/source/t/topaz-em/topaz-em-0.2.5.tar.gz"`) to the older version directive.